src/index.js

```diff
@@ -122,7 +131,56 @@ class SimpleImage {
    * @public
    */
   render() {
-    let wrapper = this._make('div', [this.CSS.baseClass, this.CSS.wrapper]),
+    this.wrapper = this._make('div', [this.CSS.baseClass, this.CSS.wrapper]);
+    if(this.data && this.data.url.length) {
+      this._createImage();
+    } else {
+      this.createFileButton();
+      const imageDataPresent = this.eventTypes.find(e => e === event.type);
+      if(!imageDataPresent) {
+        this.uploadFile.click();
+      }
+    }
+    return this.wrapper;
+  }
+
+  createFileButton() {
+    const uploadButton = this._make('div', ['cdx-button']);
+    uploadButton.innerHTML = `${this.buttonIcon} Select an Image`;
+    this.uploadFile = this._make('input');
+    this.uploadFile.setAttribute('type', 'file');
+    this.uploadFile.setAttribute('id', 'uploadedFile');
+    this.uploadFile.setAttribute('accept', 'image/*');
+    uploadButton.appendChild(this.uploadFile);
+    uploadButton.addEventListener('click', this.uploadButtonClick);
```

Comment:
why we need this listener?

Reply:
now i have hidden the file input and showed a fileupload button, user clicks this button on which events for file-input is triggered programatically